content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst

```diff
@@ -101,7 +101,7 @@ date on which this draft bill should be billed.
 
 Then, under the :guilabel:`Invoice Lines` tab, click :guilabel:`Add a line`, and click the drop-down
 menu in the :guilabel:`Product` column to select the previously created landed cost product. Click
-the :guilabel:`Save manually (cloud with arrow)` icon to update the draft bill.
+the :icon:`fa-cloud-upload` :guilabel:`(cloud)` icon to update the draft bill.
```

Comment:
```suggestion
the :icon:`fa-cloud-upload` :guilabel:`(cloud with arrow)` icon to manually save and update the draft bill.
```
SUGGESTION: adding the specificity of the guilabel just in case the reader is looking for **only** a cloud icon...and also, implying what the function of that icon does by adding "...to manually save..." in the remainder of the sentence.